app/src/main/java/grinnell/appdev/edu/lyles/menuItems/BeerItem.java

```diff
@@ -1,16 +1,18 @@
 package grinnell.appdev.edu.lyles.menuItems;
 
 /**
- * Model beer item. Contains
+ * Model beer item. Contains information on beer name and beer description
```

Comment:
Good that you have the javadoc here. The first line of your comment should be what the class does. Please put all the explanation for the description on the next line